src/pages/login/login.html

```diff
@@ -10,28 +10,38 @@
 <ion-content>
   <form (submit)="login()">
     <ion-list>
+      <ion-item>
+        <ion-label fixed>Org Name</ion-label>
+        <ion-input type="text" name="OrgName"></ion-input>
+      </ion-item>
 
       <ion-item>
-        <ion-label fixed>{{ 'EMAIL' | translate }}</ion-label>
-        <ion-input type="email" name="email"></ion-input>
+        <ion-label fixed>Tax Id </ion-label>
+        <ion-input type="text" name="taxId"></ion-input>
       </ion-item>
 
-      <!--
-      Want to use a Username instead of an Email? Here you go:
+      <ion-item>
+        <ion-label fixed>Address </ion-label>
+        <ion-input type="text" name="orgAddress"></ion-input>
```

Comment:
I would break this down even more into "street address" "city" "state" "zip code"